fetch_moveit_config/launch/fetch_moveit_sensor_manager.launch.xml

```diff
@@ -1,12 +1,12 @@
 <launch>
 
-  <!-- Set the file describing which sensor(s) to use to build the octomap  -->
-  <arg name="moveit_octomap_sensor_params_file" default="$(find fetch_moveit_config)/config/sensors.yaml" />
-
   <!-- Params for the octomap monitor -->
   <param name="octomap_frame" type="string" value="base_link" />
   <param name="octomap_resolution" type="double" value="0.05" />
 
+  <!-- Set the file describing which sensor(s) to use to build the octomap  -->
```

Comment:
Why move this down a few lines?

Reply:
So `octomap_resolution` can be overridden as well.